BLSQ/iaso
# POLIO-1315: Download scopes button is available to non admin users

Explain what problem this PR is resolving
- The download scope button was available to user which are not admin
- The ticket is about to avoir a user not admin to access the download scope svp from the calendar

Related JIRA tickets : [POLIO-1315](https://bluesquare.atlassian.net/browse/POLIO-1315)

## Self proofreading checklist


## Changes

- Check if the current user has iaso_polio_config permission before displaying the download scope csv button

## How to test
- Create or edit a user by removing the iaso_polio_config permission
- Go to calendar and check if the user has access to the csv download scope


[POLIO-1315]: https://bluesquare.atlassian.net/browse/POLIO-1315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
